### PR TITLE
Add stackdriver monitoring option

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**

### CODE REVIEW

**Comment by piosz** [comment]
Please update Heaspter to `v1.3.0`. The previous version `v1.3.0-beta.1` doesn't have stackdriver sink. And the new one doesn't support `-disable_export` flag so you need to remove it and add TODO to introduce it with the next version bump.

Then please test that the newly introduced configuration actually works.

File: cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            - --sink=stackdriver
+              # TODO: add --disable_export when it's merged into Heapster release
+          volumeMounts:
+            - name: ssl-certs
+              mountPath: /etc/ssl/certs
+              readOnly: true
+            - name: usr-ca-certs
+              mountPath: /usr/share/ca-certificates
+              readOnly: true
+        - image: gcr.io/google_containers/heapster-amd64:v1.3.0
```
Comment (piosz) [inline cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml]: we do not want to have eventer here

**Review by piosz — CHANGES_REQUESTED** [review]

File: cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml (hunk 1 above)
Comment (aleksandra-malinowska) [inline cluster/addons/cluster-monitoring/stackdriver/heapster-controller.yaml]: Removed

**Review by piosz — APPROVED** [review]

**Pull request merged**